back/scripts/utils/datagouv_api.py

```diff
@@ -0,0 +1,140 @@
+import json
+import logging
+from itertools import chain
+from pathlib import Path
+from typing import Tuple
+
+import pandas as pd
+
+from back.scripts.loaders.base_loader import retry_session
+
+
+def datagouv_page_content(url: str, params: dict | None = None) -> Tuple[dict, str]:
```

Comment:
```suggestion
def _get_next_page(url: str, params: dict | None = None) -> Tuple[dict, str]:
```
clearer?
also let's make it private, and move bellow public methods?